tools/reproduce_comparison/Jenkinsfile

```diff
@@ -14,27 +14,34 @@ pipeline {
     stages {
         stage('Lets Compare') {
             agent {
-                label "${params.platform}"
-            }
+                label "${params.platform}&&build"
+            } 
             steps {
                 script {
-                    cleanWs()
                     try {
                         echo "Fetching artifact1 from ${params.URL1}"
-                        sh "mkdir url1Dir1 && wget -q ${params.URL1} -O - | tar -xz -C url1Dir1"
+                        def ret1 = sh returnStatus: true, script: "mkdir url1Dir1 && wget -q ${params.URL1} -O - | tar -xz -C url1Dir1"
                         echo "Fetching artifact2 from ${params.URL2}"
-                        sh "mkdir url1Dir2 && wget -q ${params.URL2} -O - | tar -xz -C url1Dir2"
+                        def ret2 = sh returnStatus: true, script: "mkdir url1Dir2 && wget -q ${params.URL2} -O - | tar -xz -C url1Dir2"
+                        if (ret1 || ret2) { 
+                            currentBuild.result = "UNSTABLE"
```

Comment:
recommend adding more logging so we know which of the two failed.

Reply:
so the step "sh" prints out error already, e.g 
```
+ mkdir url1Dir2
+ wget -q https://ci.adoptopenjdk.net/job/build-scripts/job/openjdk18-pipeline/256/artifact/target/linux/x64/temurin/OpenJDK18U-jdk_x64_linux_hotspot_2022-09-03-23-30.tar.gz -O -
+ tar -xz -C url1Dir2

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```